packages/react-native/Libraries/EventEmitter/RCTDeviceEventEmitter.d.ts

```diff
@@ -16,7 +16,7 @@ import EventEmitter, {
  * Deprecated - subclass NativeEventEmitter to create granular event modules instead of
  * adding all event listeners directly to RCTDeviceEventEmitter.
  */
-interface DeviceEventEmitterStatic extends EventEmitter {
+export default interface DeviceEventEmitterStatic extends EventEmitter {
   sharedSubscriber: EventSubscriptionVendor;
```

Comment:
`sharedSubscriber` doesn't exist (anymore). Can you remove it?

Reply:
Done!